site/src/sphinx/client-retrofit.rst

```diff
@@ -22,7 +22,7 @@ you get the following benefits:
 .. code-block:: java
 
     import com.linecorp.armeria.client.Clients;
-    import com.linecorp.armeria.client.HttpClient;
+    import com.linecorp.armeria.client.AsyncHttpClient;
```

Comment:
This import seems unused. Could you check?

Reply:
Oops, don't know why it was there.